test/functional/http/get/index.js

```diff
@@ -1,10 +1,50 @@
-require('./accounts');
-require('./blocks');
-require('./dapps');
-require('./delegates');
-require('./loader');
-require('./multisignatures');
-require('./multisignatures.post');
-require('./node');
-require('./peers');
-require('./transactions');
+'use strict';
+
+var cluster = require('cluster');
+var child_process = require('child_process');
+
+var paths = [
+	'./accounts',
+	'./blocks',
+	'./dapps',
+	'./delegates',
+	'./loader',
+	'./multisignatures',
+	'./multisignatures.post',
+	'./node',
+	'./peers',
+	'./transactions',
+];
+
+var parallelTestsRunning = {};
+
+paths.forEach(function (test) {
```

Comment:
The code responsible for running tests parallel should be reused as it is done now in 3 places